lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -511,31 +509,22 @@ eventually = eventuallyUsingDelay (500 * ms)
 -- It sleeps for a specified delay between retries.
 eventuallyUsingDelay
     :: Int -- ^ Delay in microseconds
+    -> String -- ^ Brief description of the IO action
     -> IO a
     -> IO a
-eventuallyUsingDelay delay io = do
+eventuallyUsingDelay delay desc io = do
     winner <- race (threadDelay $ 180 * oneSecond) trial
     case winner of
         Left _ -> fail
-            "waited more than 3min for action to eventually resolve."
+            ("waited more than 3min for action to eventually resolve.\
+            \ Action: " ++ show desc)
         Right a ->
             return a
   where
     trial = io `catch` \(_ :: SomeException) -> do
         threadDelay delay
         trial
 
-eventually_ :: IO () -> IO ()
-eventually_ = eventuallyUsingDelay_ (500 * ms)
```

Comment:
What's the reason for getting rid of this one :thinking: ?

Reply:
It appears that you can use `eventually` like this:

```
eventually "xyz" $ do
  ...
```
or
```
r <- eventually "xyz" $ do
   ...
```
so I figured `eventually_` is not needed any longer.